javascript/builtins/PerformanceTiming.json

```diff
@@ -0,0 +1,1001 @@
+{
+  "javascript": {
```

Comment:
This needs to be put under the API tree:
```
"api:" {
  "PerformanceTiming": {
    ... 
  }
}
```
The file should be in `api/PerformanceTiming.json`. 
The `javascript/builtins/` folder only contains ECMAScript built-ins which are available to the Web and to nodejs. 
The `api/` folder contains Web APIs and `PerformanceTiming` is one of them.

Reply:
✅